web/src/api/fbaAPI.ts

```diff
@@ -71,12 +92,35 @@ export async function getHFIThresholdsFuelTypes(
   run_type: RunType,
   for_date: string,
   run_datetime: string
-): Promise<FireZoneThresholdFuelTypeResponse[]> {
+): Promise<Record<number, HfiThresholdFuelTypeArea[]>> {
```

Comment:
We can can delete this call now, the endpoint has been replace with the zone parameter'd one